Configure.ps1

```diff
@@ -153,31 +161,19 @@ so that they are consistent with the Applications parameters
                             -tenantId $tenantId `
                             -audience $serviceAppIdURI
 
-    # prepare the clean-up
-    $applicationsInformation = @{serviceApp=$serviceApplication; serviceAppServicePrincipal=$serviceservicePrincipal; clientApp=$daemonApplication; clientAppServicePrincipal=$daemonservicePrincipal; certificateThumbPrint=$certBase64Thumbprint}
-    return $applicationsInformation
+    # Completes
+    Write-Host "Done."
    }
 }
 
-
```

Comment:
The Cleanup function was moved to the new Cleanup.ps1 script (and much improved)